src/index.js

```diff
@@ -17,7 +17,8 @@ exports.addPrefix = (multicodecStrOrCode, data) => {
       throw new Error('multicodec not recognized')
     }
   }
-  return Buffer.concat([pfx, data])
+  let encodedPrefix = new Buffer(varint.encode(bufferToNumber(pfx)))
```

Comment:
yes we can do that but then when we have to reverse this when searching for the codec here
https://github.com/kumavis/js-multicodec/blob/ee554e538767a0da12d357b1368cce668c60be4c/src/index.js#L36

which would you prefer?
1. format name -> var-int encoded code table only, with lookups on getCodec
2. format name -> code table only, with lookups on addPrefix
3. both format name -> code table and format name -> var-int encoded code table
4. both format name -> code table and code -> var-int encoded code table